presto-native-execution/presto_cpp/presto_protocol/special/TableWriterNode.cpp.inc

```diff
@@ -16,15 +16,51 @@ TableWriterNode::TableWriterNode() noexcept {
   _type = "com.facebook.presto.sql.planner.plan.TableWriterNode";
 }
 
+void to_json(json& j, const VariableAggregation& p) {
+  j = json::object();
+  to_json_key(
+      j,
+      "variableReferenceExpression",
+      p.variableReferenceExpression,
+      "VariableAggregation",
+      "VariableReferenceExpression",
+      "variableReferenceExpression");
+  to_json_key(
+      j,
+      "aggregation",
+      p.aggregation,
+      "VariableAggregation",
+      "Aggregation",
+      "aggregation");
+
+}
+
+void from_json(const json& j, VariableAggregation& p) {
```

Comment:
Do we need these customizations? Why? What happens if we remove these files?

Reply:
Yes, because there are still column that is not included in prestissimo, for example `@JsonProperty("target") Optional<WriterTarget> target`
Inside `WriterTarget`, it has more data structures that we don't support now, my guess that `TableWriterNode.cpp.inc` existed in the first place is that we don't have a full solution for this class yet and we can remove it once we add the last piece